cluster/ci/config/prow/plugins.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: plugins
+data:
+  plugins: |
+    # Plugin repository whitelist.
+    # Keys: Full repo name: "org/repo".
+    # Values: List of plugins to run against the repo.
+    ---
+    openshift/origin:
```

Comment:
Can we draft an email explaining the workflows for these and how they're different than today?